modules/ROOT/pages/manage-client-apps-latest-task.adoc

```diff
@@ -3,12 +3,20 @@ ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-In Exchange, you can make information about APIs you develop in Anypoint Platform publicly available for applications that want to consume the API. You can limit access to certain users.
+In Exchange, you can make information about APIs you develop in Anypoint Platform publicly available for applications that want to consume the API. You can limit access to certain users. +
```

Comment:
This is unnecessarily verbose and throws information without context. 
I wouldn't mention that 

> You can limit access to certain users

without showing the reader how to do that. 

Try restructuring this introductory paragraph.